lib/signs/bus.ex

```diff
@@ -121,6 +124,23 @@ defmodule Signs.Bus do
     {:ok, state}
   end
 
+  @impl true
+  def handle_info({:play_pa_message, pa_message}, sign) do
+    Logger.info("pa_message: action=send id=#{pa_message.id} destination=#{sign.id}")
```

Comment:
The logging looks like it could be cleaned up a little. This one should probably be moved into `send_pa_message` and only done if we actually play the message. There's another log with the same `action` value in the engine, which we could remove.